asmcli/asmcli

```diff
@@ -2717,6 +2797,8 @@ FLAGS:
   --version
 EOF
 }
+MANAGED_CNI="${OPTIONS_DIRECTORY}/cni-managed.yaml"; readonly MANAGED_CNI;
```

Comment:
is there a reason for this var to be here instead of with all other global readonly vars in `init()`?

Reply:
Mostly because I don't think it actually needs to be global so I wanted to scope it to the file, but I'm not confident enough to move it to a local var yet.